app/helpers/bag.py

```diff
@@ -50,50 +50,83 @@
 from app.helpers.sidecar import Sidecar
 from app.services.org_api import OrgApiClient
 
+# Library of Congress Recommended Formats Statement
+AUDIO_MEDIA_INDEPENDENT = "Audio - Media-independent (digital)"
+COLLECTION = "Collection"
+OTHER_GRAPHIC_IMGS_DIGITAL = "Other Graphic Images - Digital"
+PHOTOGRAPHS_DIGITAL = "Photographs - Digital"
+TEXTUAL_DIGITAL = "Textual works - Digital"
+VIDEO_FILE_BASED = "Video - File-based and Physical Media"
+
+# Hardcoded mimetypes mapping to avoid non-deterministic results via the Python
+# mimetypes module which is based on the OS-specific `/etc/mime.types`.
 EXTENSION_MIMETYPE_MAP = {
-    ".jpg": "image/jpeg",
-    ".pdf": "application/pdf",
-    ".tiff": "image/tiff",
-    ".tif": "image/tiff",
-    ".mxf": "application/mxf",
-    ".mov": "video/quicktime",
-    ".mp4": "video/mp4",
-    ".mp3": "audio/mpeg",
-    ".wav": "audio/x-wav",
+    ".ac3": "audio/ac3",
+    ".aif": "audio/aiff",
+    ".avi": "video/x-msvideo",
+    ".dng": "image/x-adobe-dng",
+    ".dv": "video/x-dv",
+    ".f4v": "video/mp4",
+    ".flv": "video/x-flv",
     ".jp2": "image/jp2",
     ".jpeg": "image/jpeg",
+    ".jpg": "image/jpeg",
+    ".m2v": "video/mpeg",
+    ".m4a": "audio/mp4",
+    ".m4v": "video/mp4",
+    ".mkv": "video/x-matroska",
+    ".mov": "video/quicktime",
     ".mp2": "audio/mpeg",
+    ".mp3": "audio/mpeg",
+    ".mp4": "video/mp4",
+    ".mpeg": "video/mpeg",
     ".mpg": "video/mpeg",
+    ".mts": "video/MP2T",
+    ".mxf": "application/mxf",
     ".ogg": "audio/ogg",
-    ".zip": "application/zip",
+    ".pdf": "application/pdf",
+    ".png": "image/png",
+    ".psb": "image/vnd.adobe.photoshop",
```

Comment:
Ah, moet het "image/vnd.adobe.photoshop" zijn? Ik heb de waarde van die mimetype uit MediaHaven gehaald maar die is dan niet juist?

Reply:
Seems that it should be `image/vnd.adobe.photoshop`:
- https://www.iana.org/assignments/media-types/media-types.xhtml
- https://mimetype.io/image/vnd.adobe.photoshop.

Reply:
`image/vnd.adobe.photoshop` used in  `.psd`. Different file though but `image/vnd.adobe.photoshop` is fine by me.